src/components/sbb-card/sbb-card.scss

```diff
@@ -0,0 +1,130 @@
+// Default component properties, defined for :host. Properties which can not
+// travel the shadow boundary are defined through this mixin
+@include host-component-properties;
+
+:host {
+  --sbb-card-padding-inline: var(--sbb-spacing-responsive-xxs);
+  --sbb-card-padding-block-start: var(--sbb-spacing-fixed-3x);
+  --sbb-card-padding-block-end: var(--sbb-card-padding-block-start);
+
+  position: relative;
+  overflow: hidden;
+  border-radius: var(--sbb-border-radius-4x);
+  transition: box-shadow var(--sbb-animation-duration-6x) var(--sbb-animation-easing);
+  z-index: 0;
+
+  &::before,
+  &::after {
+    position: absolute;
+    content: '';
+    inset: 0;
+  }
+
+  &::before {
+    background-color: var(--sbb-color-white-default);
+    z-index: -3;
+  }
+
+  &::after {
+    background-color: transparent;
+    z-index: -1;
+  }
+
+  @include ifForcedColors {
+    outline: var(--sbb-border-width-1x) solid CanvasText;
+  }
+}
```

Comment:
Do we need an outline on focus-visible ?

Reply:
I added this because there's no delimitation for the card, but I could remove/change it. Would you suggest some other solution (border,..?) ?
<img width="842" alt="image" src="https://user-images.githubusercontent.com/101575400/191288361-47fb4784-94bd-4fde-8e6c-88c8ed5bd9bc.png">

Reply:
There's nothing in Figma.. @mcilurzo what do you think?

Reply:
discussed with @mcilurzo 

please add focus outline as in other components